product_archive_sold_product/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Product Archive Sold Product",
+    "version": "15.0.1.0.0",
+    "author": "Quartile Limited",
+    "website": "https://www.quartile.co",
+    "category": "Extra Tools",
+    "license": "LGPL-3",
+    "depends": ["sale"],
```

Comment:
```suggestion
    "depends": ["sale_stock"],
```
Not strictly a technical dependency, but product type `product` comes from stock module.